src/backend/marsha/core/models/video.py

```diff
@@ -149,6 +150,21 @@ class Video(BaseFile):
         help_text=_("Is the video publicly accessible?"),
     )
 
+    tags = ArrayField(
```

Comment:
This raises 2 questions:
- how is translation handled?
- can we manage tags dynamically?

I'm wondering if we need to manage tags in a dedicated table.

Reply:
I don't think tags should be translated. What do you mean by managing tags dynamically ?

Reply:
> Also, note that ArrayField does not work with mysql.

AFAIK we didn't plan to use mysql. IMO it's a bad idea to use it.

Reply:
discssed IRL and agreed :+1: